lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -911,79 +889,6 @@ sub send_to_beta :Chained('base') :PathPart('send_to_beta') :Args(0) {
     return $c->forward($c->view('JSON'));
 }
 
-sub asana :Chained('base') :PathPart('asana') :Args(0) {
```

Comment:
It sounds like we don't really know what we want the asana integration behaviour to be, so the code might change as well in the future, even if it remains in the controller. Not sure